src/ApiRoute.php

```diff
@@ -260,7 +260,17 @@ public function match(Nette\Http\IRequest $httpRequest): ?array
 
 		$url = $httpRequest->getUrl();
 
-		$path = $url->getPath();
+		// Resolve base path
+		$basePath = $url->getBasePath();
+		if (strncmp($url->getPath(), $basePath, strlen($basePath)) !== 0) {
+			return null;
+		}
+		$path = substr($url->getPath(), strlen($basePath));
+
+		// Ensure start with /
+		if ($path === '' || $path[0] !== '/') {
+			$path = '/' . $path;
+		}
```

Comment:
Hmm, maybe i miss somethink, but your code remove left and right /, prev code just add left one.

Reply:
Yep. My change just added slash at the begging. It's because of replacing basePath with is also at least `/` or `/folder/`. So the slash at the end (begging of new path) is missing. 

- `` => `/`
- `users` => `/users`
- `users/` => `/users/`

Reply:
Good point, updated (`trim` -> `ltrim`)